src/scythe-bidder/faction-mat-combination.tsx

```diff
@@ -57,12 +57,35 @@ class FactionMatCombination extends React.Component<Props> {
   }
 
   getFactionIcon() {
+    const faction = this.props.combination.faction;
+    let imageLocation = "";
```

Comment:
Same comment as above: can we use let imageLocation = `./static/images/${faction}.webp`; instead?

Reply:
So two of the faction icons are webp and the other 5 are png which is why I felt I needed the switch block right below that